src/applications/ivc-champva/10-10D/components/Sponsor/sponsorAlerts.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+const blankSchema = { type: 'object', properties: {} };
+
+export const homelessInfo = {
+  uiSchema: {
+    'view:homelessInfo': {
+      'ui:description': (
+        <va-alert status="info" visible uswds>
+          <h3 slot="headline">Resources available</h3>
+          <p>
+            If you don’t currently have a mailing address or are experiencing
+            homelessness you can use{' '}
+            <a href="https://faq.usps.com/s/article/What-is-General-Delivery">
```

Comment:
can this be a `va-link` component? Also, not sure if this should include `target=_blank` or not, since this is  part of a form flow.

Reply:
Changed to va-link and added target=_blank

Reply:
Actually, looking into it - seems like the va-link doesn't actually support the target property. From an accessibility standpoint, is it better to go back to a regular <a>? 

[reference](https://design.va.gov/storybook/?path=/docs/components-va-link--default)